app/controllers/review_controller.rb

```diff
@@ -95,4 +96,8 @@ def generate_change_note(status, change_note = nil)
   def set_step_by_step_page
     @step_by_step_page = StepByStepPage.find(params[:step_by_step_page_id])
   end
+
+  def require_user_to_be_the_2i_reviewer!
+    head :forbidden unless current_user.uid == @step_by_step_page.reviewer_id
```

Comment:
I assume this means a user gets a blank response? can we can show the users an error page.

Reply:
Good spot. I'll [reuse the Content Publisher implementation](https://github.com/alphagov/content-publisher/search?q=forbidden&unscoped_q=forbidden) as best I can.